install-server.sh

```diff
@@ -71,6 +71,10 @@ check_deps() {
         echo "unzip is required by this script, please install it and try again."
         exit 1
     fi
+    if (( ! which aa-status >/dev/null 2>&1 && ! which apparmor_parser >/dev/null 2>&1 )); then
```

Comment:
I have tried this, and (apart of having a syntax error that is fixed removing the parenthesis from the comparison) in my case even though having `apparmor-parser` installed (by zypper in my case), these 2 binaries are not found by `which`.

Reply:
Update: apparmor-parser is there but in `/sbin/apparmor_parser` which was not set in my PATH, and I don't know if this is normal or not... but `aa-status` is not anywhere. But k3s works without this last utility

Reply:
`aa-status` comes in the `apparmor-utils` package

Reply:
Oh hell ok, then I have to find a cooler way to see if the system is apparmor-enabled :(